libnavigation-core/src/main/java/com/mapbox/navigation/core/adasis/AdasisConfigPathsConfigs.kt

```diff
@@ -0,0 +1,60 @@
+package com.mapbox.navigation.core.adasis
+
+import com.mapbox.navigation.base.ExperimentalPreviewMapboxNavigationAPI
+
+/**
+ * ADASISv2 path level specific configurations
+ *
+ * @param mpp most probable path config
+ * @param level1 level 1 path config
+ * @param level2 level 2 path config
+ */
+@ExperimentalPreviewMapboxNavigationAPI
+class AdasisConfigPathsConfigs(
```

Comment:
internal constructor + builder
Otherwise it'll be more difficult to extend the API.
But probably also not critical for the snapshot.